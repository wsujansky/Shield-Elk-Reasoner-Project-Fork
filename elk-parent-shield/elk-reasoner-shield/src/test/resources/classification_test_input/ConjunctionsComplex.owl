Prefix( : = <http://example.org/> )
Ontology(

SubClassOf(:A :B) 
SubClassOf(:A :C)
SubClassOf(:A :D)

SubClassOf(ObjectIntersectionOf(:B :B) :BB)
SubClassOf(ObjectIntersectionOf(:C :C) :CC)
SubClassOf(ObjectIntersectionOf(:D :D) :DD)

SubClassOf(ObjectIntersectionOf(:B :C) :BC)
SubClassOf(ObjectIntersectionOf(:B :D) :BD)
SubClassOf(ObjectIntersectionOf(:C :B) :CB)
SubClassOf(ObjectIntersectionOf(:C :D) :CD)
SubClassOf(ObjectIntersectionOf(:D :C) :DC)
SubClassOf(ObjectIntersectionOf(:D :B) :DB)

SubClassOf(ObjectIntersectionOf(:B :C :D) :BCD)
SubClassOf(ObjectIntersectionOf(:B :D :C) :BDC)
SubClassOf(ObjectIntersectionOf(:C :B :D) :CBD)
SubClassOf(ObjectIntersectionOf(:C :D :B) :CDB)
SubClassOf(ObjectIntersectionOf(:D :B :C) :DBC)
SubClassOf(ObjectIntersectionOf(:D :C :B) :DCB)

)