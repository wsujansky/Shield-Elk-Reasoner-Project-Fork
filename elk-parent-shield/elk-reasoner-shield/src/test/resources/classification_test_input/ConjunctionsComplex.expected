Ontology(
Declaration(Class(<http://example.org/A>))

Declaration(Class(<http://example.org/B>))
Declaration(Class(<http://example.org/C>))
Declaration(Class(<http://example.org/D>))

Declaration(Class(<http://example.org/BB>))
Declaration(Class(<http://example.org/CC>))
Declaration(Class(<http://example.org/DD>))

Declaration(Class(<http://example.org/BC>))
Declaration(Class(<http://example.org/BD>))
Declaration(Class(<http://example.org/CB>))
Declaration(Class(<http://example.org/CD>))
Declaration(Class(<http://example.org/DC>))
Declaration(Class(<http://example.org/DB>))

Declaration(Class(<http://example.org/BCD>))
Declaration(Class(<http://example.org/BDC>))
Declaration(Class(<http://example.org/CBD>))
Declaration(Class(<http://example.org/CDB>))
Declaration(Class(<http://example.org/DBC>))
Declaration(Class(<http://example.org/DCB>))

SubClassOf(<http://example.org/A> <http://example.org/B>)
SubClassOf(<http://example.org/A> <http://example.org/C>)
SubClassOf(<http://example.org/A> <http://example.org/D>)

SubClassOf(<http://example.org/B> <http://example.org/BB>)
SubClassOf(<http://example.org/C> <http://example.org/CC>)
SubClassOf(<http://example.org/D> <http://example.org/DD>)

SubClassOf(<http://example.org/A> <http://example.org/BC>)
SubClassOf(<http://example.org/A> <http://example.org/BD>)
SubClassOf(<http://example.org/A> <http://example.org/CB>)
SubClassOf(<http://example.org/A> <http://example.org/CD>)
SubClassOf(<http://example.org/A> <http://example.org/DC>)
SubClassOf(<http://example.org/A> <http://example.org/DB>)

SubClassOf(<http://example.org/A> <http://example.org/BCD>)
SubClassOf(<http://example.org/A> <http://example.org/BDC>)
SubClassOf(<http://example.org/A> <http://example.org/CBD>)
SubClassOf(<http://example.org/A> <http://example.org/CDB>)
SubClassOf(<http://example.org/A> <http://example.org/DBC>)
SubClassOf(<http://example.org/A> <http://example.org/DCB>)

)